apps/utils.py

```diff
@@ -77,3 +81,35 @@ def default(self, o):  # pylint: disable=method-hidden
         if isinstance(o, decimal.Decimal):
             return float(o)
         return super(DecimalEncoder, self).default(o)
+
+
+def get_s3_client():
+    if settings.ENVIRONMENT in ['LOCAL', 'TEST']:
+        s_3 = boto3.client(
+            's3',
+            endpoint_url=settings.SCHEMA + settings.S3_HOST,
+            aws_access_key_id=settings.AWS_SERVER_KEY,
+            aws_secret_access_key=settings.AWS_SERVER_SECRET_KEY,
+            config=Config(signature_version='s3v4'),
+            region_name='us-east-1'
+        )
+
+        s3_resource = boto3.resource(
+            's3',
+            endpoint_url=settings.SCHEMA + settings.S3_HOST,
+            aws_access_key_id=settings.AWS_SERVER_KEY,
+            aws_secret_access_key=settings.AWS_SERVER_SECRET_KEY,
+            config=Config(signature_version='s3v4'),
+            region_name='us-east-1'
+        )
+    else:
+        session = boto3.Session(
+            aws_access_key_id=settings.AWS_SERVER_KEY,
```

Comment:
For env != LOCAL, the S3 keys will be set automatically, no need to specify them